cherrypy/test/test_states.py

```diff
@@ -161,6 +161,14 @@ def test_1_Restart(self):
         self.assertEqual(db_connection.running, False)
         self.assertEqual(len(db_connection.threads), 0)
 
+    @pytest.mark.forked
+    @pytest.mark.xfail(
+        bool(os.getenv('CI')),
+        reason=(
+            'fails on Windows because of pytest-dev/pytest-forked#44 and '
```

Comment:
The goal is to prevent "forked" from running on Windows, otherwise, it just fails the whole test run. You need to apply one of the workarounds described @ https://github.com/pytest-dev/pytest-forked/issues/44. Otherwise, it'll keep crashing pytest itself in the CI: https://ci.appveyor.com/project/CherryPy/cherrypy/builds/36337002/job/2cq5o4w0e87bnk12/tests